scripts/golangci.yml

```diff
@@ -0,0 +1,153 @@
+---
+# https://github.com/golangci/golangci-lint/blob/master/.golangci.example.yml
```

Comment:
would it be possible to curl this file instead of checking it in?

Reply:
the reference is taken from the linked file, and I have modified some values as per our requirement.